ui/src/dashboards/selectors/index.test.ts

```diff
@@ -118,6 +136,70 @@ describe('Dashboards.Selector', () => {
       type: 'custom',
     }
 
+    mocked(getTimezoneOffset).mockImplementation(() => 420)
```

Comment:
~can you add a comment here saying this is the value returned for california in daylight savings time?~

actually ignore the above, it's subtly wrong. this is the value returned for -7 UTC timezone, which is the timezone california is in during daylight savings time. in the winter time, this will be 360.